src/MountedTraversal.js

```diff
@@ -49,9 +49,8 @@ export function instHasClassName(inst, className) {
   if (!isDOMComponent(inst)) {
     return false;
   }
-  let classes = findDOMNode(inst).className || '';
-  classes = classes.replace(/\s/g, ' ');
-  return ` ${classes} `.indexOf(` ${className} `) > -1;
+  let classes = findDOMNode(inst).classList;
```

Comment:
So the question is, does implementing `classList` provide any benefits over the current implementation? Since we'll have to implement a method to check `className` for SVGs as well, is there a sufficient advantage here to include a check and multiple code paths?